oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/LogoutHandlerTest.java

```diff
@@ -302,7 +302,7 @@ void shouldRedirectToDefaultLogoutUriWithErrorMessageWhenIDTokenHintIsNotFoundIn
         verify(logoutService)
                 .generateErrorLogoutResponse(
                         Optional.empty(),
-                        new ErrorObject(OAuth2Error.INVALID_REQUEST_CODE, "invalid session"),
+                        new ErrorObject(OAuth2Error.INVALID_REQUEST_CODE, "unable to validate id_token_hint"),
```

Comment:
Is invalid the same as unable to validate?

Reply:
Not sure if this is still applicable ?

Reply:
Ah sorry - wrong thing. This doesn't change the test but changing the description to match the production code so its more obvious what we are testing.

Reply:
Changed the logging back to the previous descriptions for now.

Reply:
If its possible will update in another PR

Reply:
Isaac has checked that the changing of these logs has no impact